loopd/run.go

```diff
@@ -168,7 +170,17 @@ func Start(rpcCfg RPCConfig) error {
 
 	// Execute command.
 	if parser.Active == nil {
-		return daemon(&config, lisCfg)
+		signal.Intercept()
+
+		daemon := New(&config, lisCfg)
+		if err := daemon.Start(); err != nil {
+			return err
+		}
+
```

Comment:
Discussed offline: If a daemon error happens here, the cli will hang indefinitely.

Reply:
Addressed by listening on `errChan`.